core/controllers/story_viewer.py

```diff
@@ -99,18 +99,19 @@ def post(self, story_id, node_id):
             raise self.PageNotFoundException(e)
 
         story = story_fetchers.get_story_by_id(story_id)
+        completed_nodes = []
```

Comment:
Why redeclare? Probably can omit this line and just declare it once on line 104.

Reply:
Done